presto-main/src/main/java/com/facebook/presto/sql/planner/plan/AggregationNode.java

```diff
@@ -131,7 +133,9 @@ public AggregationNode(
         outputs.addAll(getGroupingKeys());
         hashSymbol.ifPresent(outputs::add);
         outputs.addAll(assignments.keySet());
-
+        if ((step == Step.PARTIAL || step == Step.INTERMEDIATE)) {
```

Comment:
Since partial is not supported in execution, you shouldn't add order by symbols for `PARTIAL` and `INTERMEDIATE`. Maybe add `checkState` here to validate that `ORDER BY` and aggregation step is SINGLE.